tests/test_views.py

```diff
@@ -3885,6 +3885,54 @@ def test_register_logout_flage_without(self):
         assert_true(data.get('must_login_warning'))
 
 
+class TestAuthLogout(OsfTestCase):
+
+    def setUp(self):
+        super(TestAuthLogout, self).setUp()
+        self.goodbye_url = web_url_for('goodbye', _absolute=True)
+        self.redirect_url = web_url_for('forgot_password_get', _absolute=True)
+        self.valid_next_url = web_url_for('dashboard', _absolute=True)
+        self.invalid_next_url = 'http://localhost:1234/abcde'
+        self.auth_user = AuthUserFactory()
+        self.user = UserFactory()
```

Comment:
This appears unused -- each of the tests that uses any user references `self.auth_user`.